synapse_data_warehouse/synapse_raw/tasks/V2.29.1__delete_userprofile_latest_tasks.sql

```diff
@@ -0,0 +1,8 @@
+-- Drop any scheduled tasks
+USE SCHEMA {{database_name}}.synapse_raw;
+-- Suspend ROOT TASK
+ALTER TASK IF EXISTS REFRESH_SYNAPSE_WAREHOUSE_S3_STAGE_TASK SUSPEND;
+-- Drop LATEST_TABLE UPSERTING TASK
+DROP TASK IF EXISTS UPSERT_TO_USERPROFILE_LATEST_TASK;
```

Comment:
I realized that adding `IF EXISTS` is probably not the best idea because schemachange will silently not drop anything if there is a typo in the task name for example. The same goes for any other commands with the `IF EXISTS` conditional

Reply:
Will drop IF EXISTS. Thanks for catching this.